packages/eyes/src/index.ts

```diff
@@ -1,16 +1,39 @@
 /* tslint:disable:no-console */
 
+require('dotenv').config();
+const {Eyes} = require('eyes.images');
+
 import path from 'path';
 import puppeteer from 'puppeteer';
-import {WebpackConfigurator, serve, IServer, waitForPageError, logConsoleMessages} from '@ui-autotools/utils';
+import {WebpackConfigurator, serve, IServer, waitForPageError, logConsoleMessages, consoleLog} from '@ui-autotools/utils';
+import chalk from 'chalk';
+import uuid from 'uuid';
 
 const packagePath = path.resolve(__dirname, '..');
 const projectPath = process.cwd();
 const webpackConfigPath = path.join(projectPath, 'meta.webpack.config.js');
 
-function getWebpackConfig() {
+const projectName = require(path.join(projectPath, 'package.json')).name;
+const batchName   = projectName;
+const batchId     = uuid.v4();
+const eyesApiKey  = process.env.EYES_API_KEY;
+const osName      = process.platform;
+
+if (!eyesApiKey) {
+  process.exitCode = 1;
+  throw new Error('"EYES_API_KEY" needs to be defined in a ".env" file located in the root.');
```

Comment:
just say that this environment variable needs to be defined, in CI it gets defined through the 
 admin console, for example. `.env` is just one of the options.